src/components/explore-projects/ExplorePojects.js

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import ImageList from "@mui/material/ImageList";
+import ImageListItem from "@mui/material/ImageListItem";
+import ProjectImageDisplay from "./project-image-display/ProjectImageDisplay";
+import projectData from "../../utils/assets";
+import exploreProjectsStyles from "./exploreProjectsStyles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import exploreProjectsUtils from "./exploreProjectsUtils";
+const projectImages = projectData;
+
+exploreProjectsUtils.handleProjectImagesGrid();
+const ExploreProjects = () => {
+  // handles the visibility of the popup
+  const [visible, setVisible] = React.useState(false);
+  const [image, setImage] = React.useState();
+  const [layoutVisible, setLayoutVisible] = React.useState(false);
+
+  const projectImagesStyle = { ...exploreProjectsStyles.projectImagesStyles };
+  const ImagesContainer = { ...exploreProjectsStyles.projectImagesContainer };
+  const matches = useMediaQuery("(max-width:860px)");
+  const matches2 = useMediaQuery("(max-width:720px)");
+  const matches3 = useMediaQuery("(max-width:525px)");
+  const matches4 = useMediaQuery("(max-width:390px)");
```

Comment:
what does `matches2` mean? can we have a better naming?
not sure if these are mobile, tablet, desktop?